src/GeoPing.Api/SeedData.cs

```diff
@@ -0,0 +1,63 @@
+﻿using GeoPing.Api.Data;
+using GeoPing.Api.Models;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GeoPing.Api
+{
+    public static class SeedData
+    {
+        public static void Initialize(IServiceProvider serviceProvider)
+        {
+            var context = serviceProvider.GetRequiredService<ApplicationDbContext>();
+            context.Database.EnsureCreated();
+            if (!context.Users.Any())
```

Comment:
Users should be created via UserManager like in natural order, without direc inserting records into database!

Reply:
Done